delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/SchemasRoutesSpec.scala

```diff
@@ -56,24 +57,23 @@ class SchemasRoutesSpec
   )
   private val projectRef = project.value.ref
 
-  private val myId         = nxv + "myid"
-  private val myIdEncoded  = UrlUtils.encode(myId.toString)
-  private val myId2        = nxv + "myid2"
-  private val myId2Encoded = UrlUtils.encode(myId2.toString)
-  val payload              = jsonContentOf("resources/schema.json") deepMerge json"""{"@id": "$myId"}"""
-  val payloadNoId          = payload.removeKeys(keywords.id)
-  val schema               = SchemaGen.schema(myId, projectRef, payload)
-  val current              = SchemaGen.currentState(schema, subject = subject)
-  val payloadUpdated       = payloadNoId.replace("datatype" -> "xsd:integer", "xsd:double")
-  val schemaUpdated        = SchemaGen.schema(myId, projectRef, payloadUpdated)
+  private val myId           = nxv + "myid"
+  private val myIdEncoded    = UrlUtils.encode(myId.toString)
+  private val myId2          = nxv + "myid2"
+  private val myId2Encoded   = UrlUtils.encode(myId2.toString)
+  private val payload        = jsonContentOf("resources/schema.json") deepMerge json"""{"@id": "$myId"}"""
+  private val payloadNoId    = payload.removeKeys(keywords.id)
+  private val payloadUpdated = payloadNoId.replace("datatype" -> "xsd:integer", "xsd:double")
 
   private val (orgs, projs) =
     ProjectSetup.init(orgsToCreate = List(org), projectsToCreate = List(project.value)).accepted
 
+  private val schemaImports = SchemaImports(ResolversDummy(projs).accepted)
```

Comment:
I guess this one is ok but we will need to take care of circular dependencies

Reply:
Yep, we'll have to address this in the future.